app/src/main/java/edu/nextstep/camp/calculator/MainPresenter.kt

```diff
@@ -0,0 +1,34 @@
+package edu.nextstep.camp.calculator
+
+import com.github.dodobest.domain.InputHandler
+
+class MainPresenter(
+    private val view: MainContract.View
+) : MainContract.Presenter {
+    private val inputHandler = InputHandler()
```

Comment:
```suggestion
class MainPresenter(
    private val view: MainContract.View,
    private val inputHandler: InputHandler = InputHandler()
) : MainContract.Presenter {
```

가능하면 멤버 변수를 선언하는 것보다는, 생성자를 통하여 주입하는 것이 좋습니다.
지금은 InputHandler의 객체에 생성자가 없지만, 생성자가 필요한 경우 멤버 변수로 되어 있으면 MainPresenter 관련 테스트코드를 작성할 때 멤버 변수를 초기화하지 못하여 테스트 코드를 작성하기 어려워집니다.

생성자로 주입하게 되면, MainPresenter 관련 테스트 코드 작성 시 InputHandler 객체를 mockking해서 넣어줄 수 있기 때문에 테스트 코드를 작성하기 용이해집니다.